### PR TITLE
Card Component

### PR DESCRIPTION
**Pull request by dusariudaykumar**
#Card Component
## Responsiveness:
-Mobile[❎]
-Medium Screen[✔]
-Large Screen[✔]

### CODE REVIEW

File: Components/Card/card.css [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+.flex-col {
+  display: flex;
+  flex-direction: column;
+}
+.card-outer-container {
+  width: 400px;
+  background-color: var(--white);
+  position: relative;
+  margin: 1rem 0;
+  transition: 1s ease;
```
Comment (Chiragmodi01) [inline Components/Card/card.css]: 1s transition feels a bit long to me.

File: Components/Card/card.css [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+.card-title {
+  font-size: 1.2rem;
+  font-family: sans-serif;
+  padding: 0.2rem;
+}
+.card-rating {
+  font-size: 1.1rem;
+  margin-top: 0.6rem;
+  background-color: #ff7c03;
```
Comment (Chiragmodi01) [inline Components/Card/card.css]: Prefer using variables, if you're using them more in the file.

**Review by Chiragmodi01 — COMMENTED** [review]
- Either you can increase the width of the aside navbar or your main section's. To fill up the empty white space on the right side.
- Carbon code snippets seems messed up, as they getting out of the flow and miss-aligning themselves. And also the scrollbar looks like it's floating on right.
- Some vertical padding to price will enhance the look.

Leaving some sanity comments. Overall looks pretty good.

**Review by harshitpaliwal95 — COMMENTED** [review]
need some space in card info section 
in between heading price rating

File: Components/Card/card.css (hunk 1 above)
Comment (dusariudaykumar) [reply Components/Card/card.css]: I will change it.

**Pull request merged**